tests/performance/test_coresubset.py

```diff
@@ -31,7 +31,6 @@
 # the pylint warnings for duplicated-code. Additionally, we wrap the method/function of
 # interest in a lambda function to ensure no cached JIT code is re-used to make the test
 # fair. As a result, ignore the pylint warnings for unnecessary-lambda.
-# pylint: disable=unnecessary-lambda
```

Comment:
The corresponding `# pylint: enable=unnecessary-lambda` needs to be removed from the end of the file